private/bufpkg/bufcheck/bufbreaking/bufbreakingconfig/bufbreakingconfig.go

```diff
@@ -76,6 +76,9 @@ func NewConfigV1(externalConfig ExternalConfigV1) *Config {
 
 // ConfigForProto returns the Config given the proto.
 func ConfigForProto(protoConfig *breakingv1.Config) *Config {
+	if protoConfig == nil {
```

Comment:
The codebase generally expects callers to validate this, ie if you have a pointer you are passing, it is expected to exist...I would think this should be fixed at a higher level

Reply:
Let me open an issue to address this at a higher level of the code base -- I think this is a valid concern. Right now, since we're migrating and dealing with older module structures alongside new module structures that will contain the config, the nil checks have been put in at the constructor level.